client/index.jsx

```diff
@@ -8,22 +9,39 @@ class RelatedTracks extends React.Component {
       currentSong: 1,
       related: []
     }
+
+    this.updateRelated = this.updateRelated.bind(this);
+  }
+
+  updateRelated(data) {
+    this.setState({related: data});
+  }
+
+  componentDidMount() {
+    $.ajax(
+      {
+        url: '/relatedTracks/1',
+        data: {},
+        success: data => this.updateRelated(data),
+        error: err => console.log(err)
+      }
+    );
   }
 
   render() {
-    let plays = [<span>&#9658;</span>, ` 4`];
-    let likes = [<span>&#9829;</span>, ` 6`];
-    let reposts = [<span>&#10226;</span>, ` 2`];
-    let comments = [<span>&#128488;</span>, ` 3`];
+    let plays = [<span key="spanPlays">&#9658;</span>, ` 4`];
```

Comment:
why are you using template literals?

Template literals are a function invocation. You can define your own function which modifies the result! Think "Here is your song" vs "Here are your songs" with the same template literal passing it through a "check plurality" function.

if you don't use a function, the default function is used which concats.

http://mdn.io/templateliterals

anyway, you're invoking functions when double quotes will do, or just plain number